platform/commonUI/general/src/ui/TreeNodeView.js

```diff
@@ -81,13 +81,15 @@ define([
         }
 
         this.activeObject = domainObject;
-
-        if (domainObject && domainObject.hasCapability('composition')) {
-            $(this.toggleView.elements()).removeClass('no-children');
-        } else {
-            $(this.toggleView.elements()).addClass('no-children');
+        if (domainObject && domainObject.hasCapability('adapter')) {
+            var obj = domainObject.useCapability('adapter');
+            var hasComposition =  this.openmct.composition.get(obj) !== undefined;
+            if (hasComposition) {
+                $(this.toggleView.elements()).removeClass('no-children');
+            } else {
+                $(this.toggleView.elements()).addClass('no-children');
+            }
         }
```

Comment:
Very minor style point, I would put a new line after this `}`

Reply:
Done !!  @akhenry